utilities/test_suite/HIP/runTests.py

```diff
@@ -23,18 +23,48 @@
 import argparse
 import sys
 import datetime
+import shutil
 
 # Set the timestamp
 timestamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
 
-# Set the value of an environment variable
-os.environ["TIMESTAMP"] = timestamp
-
 cwd = os.getcwd()
 inFilePath1 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
 inFilePath2 = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src2')
 qaInputFile = os.path.join(os.path.dirname(cwd), 'TEST_IMAGES', 'three_images_mixed_src1')
 
+# Check if folder path is empty, if it is the root folder, or if it exists, and remove its contents
+def validate_and_remove_contents(path):
+    if not path:  # check if a string is empty
+        print("Folder path is empty.")
+        exit()
+    if path == "/*":  # check if the root directory is passed to the function
+        print("Root folder cannot be deleted.")
+        exit()
+    if os.path.exists(path):  # check if the folder exists
```

Comment:
Add space after L43 for better readability

Reply:
Done